packages/extension/src/background/alarms.ts

```diff
@@ -0,0 +1,25 @@
+import browser from "webextension-polyfill"
+
+const transactionTrackerHistoryAlarm = "core:transactionTracker:history"
+export async function refreshTransactionTrackerHistoryAlarm() {
```

Comment:
[Nit] This function doesn't seem to be used.

Reply:
Maybe keep it just for consistency